tests/code_wars/test_find_unique_number.py

```diff
@@ -0,0 +1,11 @@
+# https://www.codewars.com/kata/585d7d5adb20cf33cb000235/train/python
+
+from code_challenges.code_wars.find_unique_number import find_unique_number
+
+
+test_numbers = {1: [1, 1, 1, 2, 1, 1], 2: [0, 0, 0.55, 0, 0]}
```

Comment:
`test_numbers` is a global variable.
1) which functions need this variable.  If only one, move to inside the function that needs it.
2) if it is to remain a global, by convention this should be UPPER_SNAKE_CASE

> if it is to remain a global

In this case it shouldn't!